docs/faq.md

```diff
@@ -67,6 +67,10 @@ kubectl port-forward -n kube-system $(kubectl -n kube-system get pods --selector
 ```
 Visit [http://127.0.0.1:9000/dashboard/](http://127.0.0.1:9000/dashboard/) in your browser to view the Traefik dashboard.
 
+**Q: Does file sharing work similarly to Docker Desktop? Do I have to do any additional configuration to mount volumes to VMs?**
+
+**A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, these files are automatically shared via WSL2. We will update the community as more mounted directories are added to this list.
```

Comment:
```suggestion
**A:** Currently, the following directories are shared by default: `/Users/$USER` on macOS, `/home/$USER` on Linux, and `/tmp/rancher-desktop` on both. For Windows, all files are automatically shared via WSL2.
```

"these files" refers to the files referred to before, which are macOS and Linux specific. So I think "all files" is better.

I think it is implied that we will update the document as our product changes; it is not specific to this particular question.

Reply:
Committed as suggested, thanks!